### DISCUSSION TITLE
AGH doesn't block anymore

### DISCUSSION THREAD

**Discussion by cadsite** [AdguardTeam/AdGuardHome#6825] [Q&A]
For years AGH did a great job, but suddenly, without any changes made by me, the software doesn't block anything anymore.
I don't see what happened.
The service is turned on
![01](https://github.com/AdguardTeam/AdGuardHome/assets/107842869/a8df30f7-b25a-408f-a0fd-5d30a70b0ff1)

The blocklists are up to date and activated

![02](https://github.com/AdguardTeam/AdGuardHome/assets/107842869/f0a525b9-64a2-4ef6-b987-2a79e02d8e95)

My fritzbox goes to the Raspberry Pi on witch AGH is installed:

![03](https://github.com/AdguardTeam/AdGuardHome/assets/107842869/8d194d36-28c9-48de-94d6-afe4ba67f7c0)

If I go to a blaclisted domain on my PC I can see the domain, but I can also see this in the log:

![04](https://github.com/AdguardTeam/AdGuardHome/assets/107842869/0922082b-0157-4a80-b0e8-fcd1966ce71c)

Anybody any idea what's going wrong here?

**Comment by ppfeufer** [comment]
Set your AGH IP as the "Alternative DNS" as well in your FritzBox.

**Comment by cadsite** [comment]
As usual...
1: Have a problem
2: search for days for a solution
3: ask the question online
4: find the solution yourself minutes after asking 

So the solution?

I recently installed an app to controll AGH on my raspberry pi. I turned the service "off" on my phone.
I had an app on my ipad for month's and on this device I could turn the service off and turn it on, on the PC, but apparently the android app I have (had to be honest) I can only turn the service 'on', on the app and not on an other device.

So the solution for me:

Turn the service on, in the android app and delete that app and continue to use the great AGH. Only use the interface on the PC and the iPad.

Thanks for thinking with me, but this is solved for me.
